.github/workflows/config.yml

```diff
@@ -1,99 +1,73 @@
 name: XSOAR CI/CD
 on:
   push:
+
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      BRANCH_NAME: ${{ github.ref_name }}
+      DEFAULT_BRANCH: ${{ github.event.repository.default_branch }}
+      DEMISTO_README_VALIDATION: false
+      DEMISTO_SDK_GITHUB_TOKEN: ${{ secrets.DEMISTO_SDK_GITHUB_TOKEN }}
```

Comment:
do we need it?

Reply:
@ilaner What do you mean? DEMISTO_SDK_GITHUB_TOKEN?

Reply:
yes, i'm not sure we use this variable anymore (only as a fallback if local git somehow fails, i think)